repose-aggregator/components/services/http-logging-service/http-logging-service-impl/src/main/scala/org/openrepose/core/services/httplogging/HttpLoggingContextMap.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.core.services.httplogging
+
+import java.time.Duration
+import java.util
+
+import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
+import org.openrepose.commons.utils.http.CommonHttpHeader
+import org.openrepose.commons.utils.logging.TracingHeaderHelper
+
+import scala.collection.JavaConverters._
+import scala.language.implicitConversions
+import scala.util.Try
+
+// TODO: This utility is not strictly necessary. Since JTwig allows access to
+// TODO: Java objects via the selection operator, we could pass the context
+// TODO: object itself into the template. That would help remove some redundancy
+// TODO: but may or may not improve user experience.
```

Comment:
I just wanted to call attention to this note to get opinions. I ended up going with the context map because it's probably easier for an operator to use since the names and navigation are simplified. That being said, I love less code.

Reply:
The map is definitely more user friendly, so it's the right choice.